data/json/monstergroups/lab.json

```diff
@@ -52,7 +52,9 @@
       { "monster": "mon_zombie_armored", "freq": 5, "cost_multiplier": 5 },
       { "monster": "mon_zombie_electric", "freq": 50, "cost_multiplier": 3 },
       { "monster": "mon_zombie_necro", "freq": 15, "cost_multiplier": 15 },
-      { "monster": "mon_zombie_brute_shocker", "freq": 10, "cost_multiplier": 5 }
+      { "monster": "mon_zombie_brute_shocker", "freq": 10, "cost_multiplier": 5 },
+      {"monster": "mon_lady_bug_giant", "freq": 15, "cost_multiplier": 2},
+      {"monster": "mon_mole_cricket", "freq": 20, "cost_multiplier": 3}
```

Comment:
What are they doing in a lab?

Reply:
As for mole crickets, I've looked at { "monster": "mon_nakedmolerat_giant", "freq": 45, "cost_multiplier": 3 } at a lab and decided to make crickets spawn there too. Probably they've burrowed into a lab.

When I'd been making bug's life mod, I've added ladybugs in labs without much thought and forgot to remove it. I'll do it in an hour.